guide/miscellaneous/useful-packages.md

```diff
@@ -263,3 +263,107 @@ Official documentation: [https://www.i18next.com](https://www.i18next.com)
 i18next is an internationalization-framework for JavaScript. It is beneficial to translate your bot's user-facing messages into various languages based on the server it is used in.
 
 Covering an entire use case example for internationalization would be out of this guide's scope and requires some more explanation as to how the system operates. Please refer to the official documentation linked above for an in-depth usage guide.
+
+## axios
+
+:::tip
+Axios is used to make HTTP(S) requests to remote servers. The documentation can be located [Here](https://axios-http.com/docs/intro)
+:::
+Example:
+<!-- eslint-skip -->
+```js
+const axios = require('axios')
+
+axios({
+	method: 'post',
+	url: 'https://example.com/v1',
+	headers:
+	{
+		'Content-Type:':'application/json',
+		'User-Agent':'MyBot'
+	},
+	data{
+		"item":"one"
+	}
+}).then(response =>{
+	console.log(response)
+})
+
+```
+
+:::warning
+If the Axios request fails for one or more reasons, it will fire an an uncaught error. For this reason, it is recommended that you place Axios requests inside of a "try","catch","finally" sequence to prevent your application from crashing.
```

Comment:
```suggestion
If the Axios request fails for one or more reasons, it will fire an an uncaught error. For this reason, it is recommended that you place Axios requests inside of a `try`, `catch` and `finally` sequence to prevent your application from crashing.
```